test/testenv/testenv.go

```diff
@@ -80,6 +80,9 @@ const (
 
 	// MonitoringConsolePod Montioring Console Statefulset Template
 	MonitoringConsolePod = "splunk-%s-monitoring-console-%d"
+
+	// SecretObject Secret object Template
+	SecretObject = "splunk-%s-secret"
```

Comment:
Nit: Change to SecretObjectName ?

Reply:
I use the secretObjectName in the function itself after i fill in the 'namespace'. Think it would be ok to keep as it is